text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+- Start Date: 2018-07-24
+- RFC PR: 
+- Gatsby Issue: 
+
+# Summary
+
+As a follow-up to the [Site Showcase](https://next.gatsbyjs.org/showcase/), we want to create individual & organization pages highlighting the individual community members and what they've built. 
+
+We also want to create a free Job Board highlighting opportunities for employment building Gatsby websites at agencies and companies using Gatsby!
+
+# Motivation
+
+Some goals for the different components include:
+
+#### Individuals Pages
+* Highlight contributions made by community members (plugins built, code PRs)
+* "Official" landing pages to help community members in their personal website marketing efforts
+* Allow companies interested in building Gatsby sites to find individuals nearby them who are open for Gatsby projects
+
+#### Agency Pages
+* Highlight agencies building websites on Gatsby
+* "Official" landing pages to help agencies in their marketing efforts
+* Allow companies interested in building Gatsby sites to find agencies nearby them
+* Help agencies hire for Gatsby-related positions by surfacing their postings to the community; give the community a place to look for Gatsby-related jobs.
+
+#### Company Pages
+* Display list of companies using Gatsby to demonstrate it's "enterprise-ready" and fulfills whatever use-case they're looking for (e-commerce, authentication, etc). 
```

Comment:
In a "one dataset" model, this could be pulled out reasonably easily:

```js
const logos = entities
	.filter(entity => entity.type === 'company')
	.map(entity => entity.profile_image);
```

Reply:
Agree with the one-dataset model.